cedar-lean/Cedar/Thm/Soundness.lean

```diff
@@ -46,50 +46,20 @@ theorem typecheck_policy_is_sound (policy : Policy) (env : Environment) (t : Ced
   typecheckPolicy policy env = .ok t →
   EvaluatesToBool policy request entities
 := by
-  intro h₀ h₁
-  unfold typecheckPolicy at h₁
-  cases h₂ : (typeOf (Policy.toExpr policy) ∅ env) <;>
-  rw [h₂] at h₁ <;>
-  simp at h₁
-  case ok res =>
-    cases h₃ : (res.fst ⊑ CedarType.bool BoolType.anyBool) <;>
-    rw [h₃] at h₁ <;>
-    simp at h₁
-    clear h₁ t
-    have h₁ : GuardedCapabilitiesInvariant policy.toExpr res.2 request entities ∧ ∃ (v : Value), EvaluatesTo policy.toExpr request entities v ∧ InstanceOfType v res.1 := by
-      apply type_of_is_sound (env:=env) (c₁:=∅)
-      apply empty_capabilities_invariant
-      apply h₀
-      apply h₂
-    cases h₁ with
-    | intro _ h₁ =>
-      cases h₁ with
-      | intro v h₁ =>
-        cases h₁ with
-        | intro h₁ h₄ =>
-          have h₅ : ∃ b, v = .prim (.bool b) := by
-            apply instance_of_type_bool_is_bool
-            apply h₄
-            apply h₃
-          cases h₅ with
-          | intro b h₅ =>
-            unfold EvaluatesToBool
-            exists b
-            rewrite [← h₅]
-            apply h₁
+  sorry
```

Comment:
Is it Ok to have sorry here?

Reply:
Yep, the soundness proofs are currently incomplete on the `main` branch. Proof work is primarily being done in the `lean-soundness-proof` branch.